tools/hab-channel-list-pkgs/hab-channel-list-pkgs.sh

```diff
@@ -43,8 +43,8 @@ main() {
 
   # Fetch each page and output a fully qualified package identifier, one per
   # line
-  for (( n=0; n<$num_pages; n+=1 )) do
-    curl -s "${url}?range=$(( ${n}*${npp} ))" \
+  for (( n=0; n<num_pages; n+=1 )) do
+    curl -s "${url}?range=$(( n * npp ))" \
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2004#-is-unnecessary-on-arithmetic-variables